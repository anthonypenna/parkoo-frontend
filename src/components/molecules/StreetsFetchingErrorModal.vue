<template>
  <Modal>
    <h2>Oh oh!</h2>
    <p>
      Seems like <strong>Parkoo</strong> couldn't connect to the
      <strong>server</strong>!
    </p>
    <p>
      We're working on it, so feel free to <strong>try again</strong> later.
    </p>
  </Modal>
</template>

<script lang="ts">
import Vue from "vue";
import Modal from "@/components/atoms/Modal.vue";

export default Vue.extend({
  name: "StreetsFetchingErrorModal",
  components: { Modal },
});
</script>
