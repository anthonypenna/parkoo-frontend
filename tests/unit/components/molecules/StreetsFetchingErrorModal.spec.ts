import { mount } from '@vue/test-utils'
import StreetsFetchingErrorModal from '@/components/molecules/StreetsFetchingErrorModal.vue'

describe('<StreetsFetchingErrorModal />', () => {
  it('should render correctly', () => {
    const wrapper = mount(StreetsFetchingErrorModal, { stubs: { Modal: true } })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
