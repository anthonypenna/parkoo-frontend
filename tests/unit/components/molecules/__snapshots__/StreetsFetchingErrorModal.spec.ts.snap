// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StreetsFetchingErrorModal /> should render correctly 1`] = `
<modal-stub>
  <h2>Oh oh!</h2>
  <p>
    Seems like <strong>Parkoo</strong> couldn't connect to the
    <strong>server</strong>!
  </p>
  <p>
    We're working on it, so feel free to <strong>try again</strong> later.
  </p>
</modal-stub>
`;
